master/proto/api/v1/agent.proto

```diff
@@ -7,35 +7,35 @@ import "agent/v1/agent.proto";
 
 // GetAgentsRequest is the request used to fetch information for a set of agents from the cluster.
 message GetAgentsRequest {
-    // SortBy sorts agents by the given field
+    // Sorts agents by the given field
     enum SortBy {
-        // SORT_BY_UNSPECIFIED returns agents sorted by id.
+        // Returns agents sorted by id.
         SORT_BY_UNSPECIFIED = 0;
-        // SORT_BY_UNSPECIFIED returns agents sorted by id.
+        // Returns agents sorted by id.
         SORT_BY_ID = 1;
-        // SORT_BY_UNSPECIFIED returns agents sorted by time.
+        // Returns agents sorted by time.
         SORT_BY_TIME = 2;
     }
-    // OrderBy orders agents in either ascending or descending order.
+    // Order agents in either ascending or descending order.
     enum OrderBy {
-        // ORDER_BY_DESC returns agents in ascending order.
+        // Returns agents in ascending order.
```

Comment:
non-blocking: I'm not sure if it makes sense to explain the default behavior?

Someone may rely on this since we claim that this is what happens. Which may burn us if we change the default behavior.